File: backend-v3/models/paymentProviderType.js
'use strict'

module.exports = (sequelize, DataTypes) => {
  var paymentProviderType = sequelize.define('paymentProviderType', {
    class: DataTypes.STRING,

    name: DataTypes.STRING,
    description: DataTypes.STRING,

    thumbnail: DataTypes.STRING,
    thumbnailGray: DataTypes.STRING,
    chartColor: DataTypes.STRING
  }, {
    freezeTableName: true,
    paranoid: true
  })

  paymentProviderType.associate = (models) => {
    paymentProviderType.hasMany(models.paymentProvider, { foreignKey: 'paymentProviderTypeId' })
  }

  return paymentProviderType
}

File: frontend-mobile-v3/data/src/main/java/id/mikaapp/data/entities/PaymentProviderType.kt
package id.mikaapp.data.entities

import com.google.gson.annotations.SerializedName

data class PaymentProviderType(
    @SerializedName("chartColor")
    var chartColor: String,
    @SerializedName("class")
    var classX: String,
    @SerializedName("description")
    var description: Any,
    @SerializedName("id")
    var id: Int,
    @SerializedName("name")
    var name: String,
    @SerializedName("thumbnail")
    var thumbnail: String,
    @SerializedName("thumbnailGray")
    var thumbnailGray: String
)